cluster-autoscaler/metrics/metrics.go

```diff
@@ -490,10 +490,10 @@ func RegisterError(err errors.AutoscalerError) {
 }
 
 // RegisterScaleUp records number of nodes added by scale up
-func RegisterScaleUp(nodesCount int, gpuType string) {
+func RegisterScaleUp(nodesCount int, resourceName, gpuType string) {
```

Comment:
nit: Gpu would be nice here and in RegisterScaleDown below as well - seems ambiguous as in executeScaleUp.

Reply:
done